ui/imports/utils/Global.qml

```diff
@@ -15,6 +15,9 @@ QtObject {
     property var mainModuleInst
     property var privacyModuleInst
     property var toastMessage
+    property var pinnedMessagesPopup
```

Comment:
hey this doesnt seem consistent throughout now, for example we pass instance of the model to the sections that needs it and then call Global.openPopup(modalComponent) and now for these 3 its different.